server/pom.xml

```diff
@@ -71,6 +71,13 @@
       <version>${spring.version}</version>
     </dependency>
 
+    <dependency>
```

Comment:
Adding this dependency as it offers certain mock objects for unit tests like `MockHttpServletResponse` which helped me a lot and avoid deeper nested mocking